src/igvfd/tests/data/inserts/primary_cell.json

```diff
@@ -23,7 +23,8 @@
         "starting_amount": 5,
         "starting_amount_units": "μg",
         "dbxrefs": [
-            "GEO:SAMN45"
+            "GEO:SAMN45",
+            "ENCODE:ENCDO123AAD"
```

Comment:
The regex pattern is ENCBS, this insert has ENCDO -- probably going to cause a test failure?

Reply:
yeah, sorry let me fix